django_date_extensions/fields.py

```diff
@@ -88,8 +96,8 @@ def __eq__(self, other):
         if not isinstance(other, ApproximateDate):
             return False
 
-        return (self.year, self.month, self.day, self.future, self.past, self.prefix) ==\
-               (other.year, other.month, other.day, other.future, other.past, other.prefix)
+        return (self.year, self.month, self.day, self.future, self.past, self.prefix, self.string_format) ==\
+               (other.year, other.month, other.day, other.future, other.past, other.prefix, self.string_format)
```

Comment:
Some extra code needs putting in `__lt__` so the code knows how to compare dates that include a `string_format`.